package.json

```
@@ -1,8 +1,16 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "lint": "yarn eslint . --ext ts,js",
+    "lint:fix": "yarn lint --fix"
+  },
   "workspaces": [
-    "packages/base"
+    "packages/base",
+    "packages/jest",
+    "packages/mocha",
+    "packages/nodejs",
+    "packages/typescript"
```

Nit: This might be more concise
```suggestion
    "packages/*",
```